README.md

```diff
@@ -93,6 +112,8 @@ confirmation prompt:
 
 _Note: this will put a secret in your command history; better use paste, see below._
 
+_Note2: when using a file based backend, provide the -pw my-master-password flag_
```

Comment:
problem with this is that the master-password will be visibile in the shell history :-(

Reply:
This is a tough one.. Not sure yet how to handle this properly

Reply:
Read it from a file from the config YAML? Something like "master-password-location", @emicklei wdyt?

Reply:
@amohabir made a change to use a proper password prompt for put and get, only for the file backend.